www/assets/js/lib/constants.ts

```diff
@@ -464,11 +457,9 @@ export const FACET_OPTIONS: Facets = {
     "Women's Studies",
     "World History"
   ],
-  type:            ["resourcefile", "course"],
-  department_name: Object.values(departments).map(
-    department => department.title
-  ),
-  level:               ["Undergraduate", "Graduate", "Non-Credit", "High School"],
-  course_feature_tags: RESOURCE_TYPES,
-  resource_type:       RESOURCE_TYPES
+  resource_type:        ["course"],
+  department:           Object.keys(departments),
```

Comment:
We should have a followup pr to get all the options from the generated api documentation